internal/daemon/controller/handlers/targets/target_service.go

```diff
@@ -268,37 +268,82 @@ func (s Service) ListTargets(ctx context.Context, req *pbs.ListTargetsRequest) (
 	}
 
 	var listResp *pagination.ListResponse[target.Target]
-	if req.GetRefreshToken() == "" {
+	sortBy := "created_time"
```

Comment:
It's a little odd to have this here, but we have to know which sort order is used, and the best indicator of that is based on which type of input token we're working with. For `StartRefreshToken` and `RefreshToken`, it's `updated_time`, otherwise it's `created_time`.

Reply:
might be better to set no default here, and just set `create_time` explicitly
where it's appropriate.

Reply:
Done